.github/ISSUE_TEMPLATE/pipelines-scrum-template.md

```diff
@@ -0,0 +1,22 @@
+---
+name: Pipelines Scrum Template
+about: 'Template for providing supplemental information on scrum cards. '
+title: ''
+labels: pipelines-scrum
+assignees: ''
+
+---
+
+## OUTCOME
+**What is the "done" condition of this card?**
+A clear and concise description of what the problem is.
```

Comment:
I think this is the outcome, so maybe - 
"A clear and concise description of the done condition of a feature, process, or task that is being capture by this issue (it may be the completion of action items defined below)."